docker-compose.yaml

```diff
@@ -2,6 +2,9 @@ version: '3'
 
 # This docker-compose is for developer convenience, not for running in production.
```

Comment:
Can you please fix the error 

time=“2025-04-14T21:28:16Z” level=warning msg=“/home/runner/work/index_runner/index_runner/docker-compose.yaml: `version` is obsolete”

Reply:
addressed